run.js

```diff
@@ -36,6 +37,27 @@ function spritesmithEngineTest(params) {
       });
     });
 
+    describe('loading buffers', function () {
+      var singleImage = config.singleImage;
+      var buffers = singleImage.filepaths.map(function(image) {
+          return {
+              id: image,
+              buffer: fs.readFileSync(image)
+          };
```

Comment:
I am not sure about using this bespoke format. I am going to continue the thread on Ensighten/spritesmith#51 for continuity.